test/acceptance/rake_task_test.rb

```diff
@@ -19,8 +19,7 @@ class RakeTaskTest < Test::Unit::TestCase
       )
       indexer.expects(:index).with([expected_recommended_link])
       indexer.expects(:remove).with([expected_deleted_link])
-      external_link_registerer.expects(:register_links)
-                              .with([expected_recommended_link])
+      external_link_registerer.expects(:register_links).never
```

Comment:
I don't think this we need to do that - ensuring that the ExternalLinkRegisterer class isn't defined in the code is a pretty thorough check of this, too.